langkit/__init__.py

```diff
@@ -29,30 +36,104 @@ class LangKitConfig:
     )
     data_folder: str = "langkit_data"
     rouge_type: str = "rouge1"
-    sentiment_lexicon: str = "vader_lexicon"
-    topic_model_path: str = "MoritzLaurer/mDeBERTa-v3-base-xnli-multilingual-nli-2mil7"
-    topic_classifier: str = "zero-shot-classification"
-    toxicity_model_path: str = "martin-ha/toxic-comment-model"
+    sentiment_lexicon: Optional[str] = "vader_lexicon"
+    response_sentiment_lexicon: Optional[str] = "vader_lexicon"
+    topic_model_path: Optional[
+        str
+    ] = "MoritzLaurer/mDeBERTa-v3-base-xnli-multilingual-nli-2mil7"
+    response_topic_model_path: Optional[
+        str
+    ] = "MoritzLaurer/mDeBERTa-v3-base-xnli-multilingual-nli-2mil7"
+    topic_classifier: Optional[str] = "zero-shot-classification"
+    response_topic_classifier: Optional[str] = "zero-shot-classification"
+    toxicity_model_path: Optional[str] = "martin-ha/toxic-comment-model"
+    response_toxicity_model_path: Optional[str] = "martin-ha/toxic-comment-model"
+    injections_transformer_name: Optional[str] = "all-MiniLM-L6-v2"
+    injections_version: Optional[str] = "v1"
 
 
 prompt_column: str = "prompt"
 response_column: str = "response"
 lang_config = LangKitConfig()
 
 
+# Override default models/parameters per language
+multi_lang_config: Dict[str, LangKitConfig] = {
+    "": LangKitConfig(),
+    "ar": LangKitConfig(
+        injections_transformer_name=None,
+        reference_corpus=None,
+        sentiment_lexicon=None,
+        topic_model_path=None,
+        response_topic_model_path=None,
+        toxicity_model_path=None,
+        response_toxicity_model_path=None,
+        transformer_name=None,
+        response_transformer_name=None,
+    ),
+    "en": LangKitConfig(),
+    "es": LangKitConfig(
+        injections_transformer_name=None,
+        reference_corpus=None,
+        sentiment_lexicon=None,
+        response_sentiment_lexicon=None,
+        topic_model_path=None,
+        response_topic_model_path=None,
+        toxicity_model_path=None,
+        response_toxicity_model_path=None,
+        transformer_name=None,
+        response_transformer_name=None,
+    ),
+    "it": LangKitConfig(
+        injections_transformer_name=None,
+        reference_corpus=None,
+        sentiment_lexicon=None,
+        response_sentiment_lexicon=None,
+        topic_model_path=None,
+        response_topic_model_path=None,
+        toxicity_model_path=None,
+        response_toxicity_model_path=None,
+        transformer_name=None,
+        response_transformer_name=None,
+    ),
+    "pt": LangKitConfig(
+        injections_transformer_name=None,
+        reference_corpus=None,
+        sentiment_lexicon=None,
+        response_sentiment_lexicon=None,
+        topic_model_path=None,
+        response_topic_model_path=None,
+        toxicity_model_path="dougtrajano/toxicity-type-detection",
+        response_toxicity_model_path="dougtrajano/toxicity-type-detection",
+        transformer_name=None,
```

Comment:
this metric throws if the transformer_name and encoder are empty, which makes this shape of language config hard to use.